lib/drivers/medtronic/medtronicDriver.js

```diff
@@ -161,7 +161,13 @@ module.exports = function (config) {
       }
       ctr += struct.pack(bytes, ctr, '7bi', 0, 0, 0, partsPerPage, 0x10, 0x10, 0x00, payload.length+secondPacketLength);
     } else {
-      ctr += struct.pack(bytes, ctr, '7bi', 0, 0, 0, 0, 0, 0, 0, payload.length);
```

Comment:
I agree. However, I can't get the number of bytes in #388 to match up with the actual packet, so I'm going to defer this to a new PR after we've had a chance to discuss.